File: Lab 3/Percentage.c
#include <stdio.h>

int main (void)
{
	int percentage, x;
	char ch1, ch2;
	scanf ("%d%c%c%d", &percentage, &ch1, &ch2, &x);
	printf ("%d%% %c%c %d is %f\n", percentage, ch1, ch2, x, 1.0 * x * percentage / 100);
  return 0;
}

File: Lab 3/Percentage.go
package main

import "fmt"

func main() {
  var percentage, x int
  var ch1, ch2 rune
  fmt.Scanf("%d%c%c%d", &percentage, &ch1, &ch2, &x)
  // NOTE: "Unlike in C, in Go assignment between items of different type
  // requires an explicit conversion." -- https://tour.golang.org/basics/13
  // fmt.Printf("%d%% %c%c %d is %f\n", percentage, ch1, ch2, x, 1.0 * x * percentage / 100)
  fmt.Printf("%d%% %c%c %d is %f\n", percentage, ch1, ch2, x, float32(x * percentage / 100))
}
